sympy/printing/maple.py

```diff
@@ -0,0 +1,182 @@
+"""
+Maple code printer
+
+The MapleCodePrinter converts single sympy expressions into single
+Maple expressions, using the functions defined in the Maple objects where possible.
+
+
+FIXME: This module is still under actively developed. Some functions may be not completed.
+"""
+
+from __future__ import print_function, division
+
+from sympy.codegen.ast import Assignment
+from sympy.core import S
+from sympy.core.compatibility import string_types, range
+from sympy.printing.codeprinter import CodePrinter
+from sympy.printing.precedence import precedence, PRECEDENCE
+
+_known_func_same_name = [
+    'sin', 'cos', 'tan', 'sec', 'csc', 'cot', 'sinh', 'cosh', 'tanh', 'sech',
+    'csch', 'coth', 'exp', 'floor'
+]
+
+known_functions = {
+    # Sympy -> Maple
+    # FIXME: maybe need added
+    'Abs': 'abs',
+    'log': 'ln',
+    'asin': 'arcsin',
+    'acos': 'arccos',
+    'atan': 'arctan',
+    'asec': 'arcsec',
+    'acsc': 'arccsc',
+    'acot': 'arccot',
+    'asinh': 'arcsinh',
+    'acosh': 'arccosh',
+    'atanh': 'arctanh',
+    'asech': 'arcsech',
+    'acsch': 'arccsch',
+    'acoth': 'arccoth',
+    'ceiling': 'ceil',
+}
+for _func in _known_func_same_name:
+    known_functions[_func] = _func
+
+atomic_expr = {
+    # Sympy -> Maple
+    S.Pi: 'Pi',
+    S.I: 'I',
+    S.Infinity: 'infinity'
+}
+
+spec_relational_ops = {
+    # Sympy -> Maple
+    '==': '=',
```

Comment:
Not sure if this will work (maybe it will). If not, the SymPy names are Eq, Ne, Ge, Gt, Le, and Lt.

Reply:
I don't think that there is any flaw with this..

Reply:
@sylee957 it just wasn't clear for me if using the string expression is a feasible way, compared to using the SymPy class name. But if it works, no problem.